src/hit/api/ciphertext.h

```diff
@@ -147,7 +147,7 @@ namespace hit {
         latticpp::Ciphertext backend_ct;
 
         // `scale` is used by the ScaleEstimator evaluator
-        double scale_ = pow(2, 30);
+        double scale_;
```

Comment:
In theory, this was ensured by carefully checking the `encrypt` procedure for each evaluator. However, it's easier to enforce with an explicit `CKKSCiphertext` constructor, so I've created one. Note, however, that we still need the empty constructor, e.g, for this snippet from my `LinearAlgebra` code:
```
CKKSCiphertext t2;
if (rotate_left) {
    t2 = eval.rotate_left(t1, i * stride);
} else {
    t2 = eval.rotate_right(t1, i * stride);
}
```
This is not a case where we want to assign to the first value and reassign to the second if the condition holds, because each assignment is expensive. But I'm using the new constructor in all `encrypt` definitions.